src/main/java/org/codehaus/plexus/archiver/zip/ConcurrentJarCreator.java

```diff
@@ -110,12 +116,14 @@ public void writeTo(ZipArchiveOutputStream targetStream) throws IOException, Exe
         metaInfDir.writeTo( targetStream );
```

Comment:
Seems like `metaInfDir` is not closed. Is it intentional? Noticed it while creating the patch so I decided to ask just in case :smile: